sci-visualization/fityk/files/fityk-1.3.1-fix-gcc11.patch

```diff
@@ -0,0 +1,18 @@
+From: Huang Rui <vowstar@gmail.com>
+Date: Sun, 9 May 2021 21:04:49 +0800
+Subject: [PATCH 1/1] use -std=c++11 to fix compile problem
+
+gcc-11 changed default cpp std
+
+Signed-off-by: Huang Rui <vowstar@gmail.com>
+
+--- a/configure.ac	2016-12-21 06:35:08.000000000 +0800
++++ b/configure.ac	2021-05-10 02:38:22.288655517 +0800
+@@ -15,6 +15,7 @@
+ m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
+ LT_INIT([disable-static win32-dll])
+ 
++AX_CHECK_COMPILE_FLAG([-std=c++11], [CXXFLAGS="$CXXFLAGS -std=c++11"])
```

Comment:
have you tried just removing the `throw(...)` specifiers?

Reply:
I thought about this method, but it will affect more than 20 files, resulting in a large diff.
https://github.com/wojdyr/fityk/search?p=1&q=throw
I guess the best way is to perform a large-scale cleanup upstream

Reply:
@SoapGentoo